tt_eager/module.mk

```diff
@@ -3,7 +3,7 @@ TT_LIBS_HOME ?= $(TT_METAL_HOME)
 TT_METAL_BASE_INCLUDES = $(BASE_INCLUDES)
 EAGER_OUTPUT_DIR = $(OUT)/dist
 
-TT_EAGER_INCLUDES = $(TT_METAL_BASE_INCLUDES) -Itt_eager/
+TT_EAGER_INCLUDES = $(TT_METAL_BASE_INCLUDES) -Itt_eager/ -I ttnn/cpp/
```

Comment:
Is this simply for convenience?

Reply:
I need functionality from ttnn in tt_eager. I think I will just move some code around. I don't want to merge it as is now